.vib/dokuwiki/cypress/cypress/integration/dokuwiki_spec.js

```diff
@@ -0,0 +1,74 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows login and logout', () => {
+  cy.login();
+  cy.get('.error').should('not.exist');
+  cy.contains('Log Out').click();
+  cy.get('#login');
+});
+
+it('allows the user to update profile', () => {
+  cy.login();
+  cy.visit('/start?do=profile');
+  cy.fixture('admins').then((admin) => {
+    cy.get('input[name="email"]')
+      .clear()
+      .type(`${admin.newAdmin.email}.${random}`);
+  });
+  cy.get('input[name="oldpass"]').first().type(Cypress.env('password'));
+  cy.contains('Save').click();
+  cy.contains('User profile successfully updated.');
+});
+
+it('allows the user to validate and modify configuration settings', () => {
+  cy.login();
+  cy.visit('start?do=admin&page=config');
+  cy.fixture('settings').then((setting) => {
+    cy.get('#config___savedir').should(
+      'contain.value',
+      setting.newSetting.saveFolder
+    );
+    cy.get('#config___dmode').should(
+      'contain.value',
+      setting.newSetting.directoryCreationMode
+    );
+  });
+  cy.fixture('settings').then((setting) => {
+    cy.get('#config___title')
+      .scrollIntoView()
+      .clear()
+      .type(setting.newSetting.newDokuWikiFullName);
+    cy.contains('button', 'Save').click();
+    cy.contains('Settings updated successfully.');
+    cy.get('.headings').should(
+      'contain',
+      setting.newSetting.newDokuWikiFullName
+    );
+  });
+});
+
+it('allows adding users', () => {
+  cy.login();
+  cy.visit('login?do=admin&page=usermanager');
+  cy.fixture('users').then((user) => {
+    cy.get('#add_userid').type(`${user.newUser.userName}.${random}`);
+    cy.get('#add_userpass').type(`${user.newUser.password}.${random}`);
+    cy.get('#add_userpass2').type(`${user.newUser.password}.${random}`);
+    cy.get('#add_username').type(`${user.newUser.realName}.${random}`);
+    cy.get('#add_usermail').type(`${user.newUser.email}.${random}`);
+  });
+  cy.contains('button', 'Add').click();
+  cy.contains('User added successfully');
+});
+
+it('allows editing a page in the playground', () => {
+  cy.login();
+  cy.visit('/playground:playground');
+  cy.get('.edit > a > svg').click();
```

Comment:
Improved the selector a bit. Since it's an icon which appears only on hover and is not supported by Cypress, I didn't have luck improving further.
I've tried some workarounds from [Cypress documentation](https://docs.cypress.io/api/commands/hover#Workarounds) but no luck.
Since this app UI doesn't seem to change often, the risk shouldn't be high.